microsoft-edge/devtools-guide-chromium/device-mode/index.md

```diff
@@ -92,6 +92,15 @@ The following table describes the differences between the available device type
 > [!NOTE]
 > If the **Device Type** list isn't displayed, select **More options** > **Add device type**.
 
+#### Add device pixel ratio
```

Comment:
in VS Code, i will examine the heading levels etc. and possibly update the usage and push a commit.